elements/lisk-p2p/src/peer/base.ts

```diff
@@ -91,7 +91,7 @@ export const REMOTE_EVENT_MESSAGE = 'remote-message';
 
 export const REMOTE_RPC_UPDATE_PEER_INFO = 'updateMyself';
 export const REMOTE_RPC_GET_NODE_INFO = 'status';
-export const REMOTE_RPC_GET_ALL_PEERS_LIST = 'list';
+export const REMOTE_RPC_GET_BASIC_PEERS_LIST = 'basicList';
```

Comment:
But we have to support `getPeers` RPC in LIP and it's doing the same job as `getPeers` is suppose to

Reply:
We still do. It is answer from the peer pool.